tensorflow_blade/src/BUILD

```diff
@@ -2,11 +2,72 @@ load("@bazel_skylib//lib:selects.bzl", "selects")
 load("@local_config_mkl//:build_defs.bzl", "if_mkl_enabled")
 load("@pybind11_bazel//:build_defs.bzl", "pybind_extension")
 load("@local_config_tf//:build_defs.bzl", "tf_copts", "tf_version")
-load("//:build_defs.bzl", "device_name")
-load("@local_config_blade_helper//:build_defs.bzl", "python_bin_path")
+load("@local_config_blade_helper//:build_defs.bzl", "if_tensorrt_enabled", "if_internal")
 
 package(default_visibility = ["//visibility:public"])
 
+##############################################tf_blade ops################################################
+# Nested select is not supported by bazel, use an intermediat target to workaround.
+cc_library(
+    name = "tensorrt_ops",
+    deps = if_tensorrt_enabled([
+        "//src/custom_ops/trt_engine_op",
```

Comment:
remove `src` folder or rename it into `tf_blade` or else. It appears in a target label but make no sence.

Reply:
The tf blade dir is reserved for python codes. Also I don't quite see the possible side effect of putting c++ code in src folder.